_editing/5_front_matter.md

```diff
@@ -222,6 +222,8 @@ styles:
 {% endraw %}
 {% endhighlight %}
 
+Don't forget the "s" character at the end of your array name.
```

Comment:
This isn't quite correct. You need to need the singlular version i.e. countries -> country